.github/workflows/pypi-release.yml

```
@@ -22,3 +22,7 @@ jobs:
         uses: pypa/gh-action-pypi-publish@master
         with:
           password: ${{ secrets.pypi_password }}
+      - name: Install cloudevents and GitPython
+        run: pip install -U cloudevents 'GitPython==3.1.7'
```

```suggestion
        run: pip install -r requirements/publish.txt
```
Why do we need to install `cloudevents` here?

---

Yeah. the git tag creation wants an installation of the most updated version of cloudevents to test if the version was successfully pushed to pypi before making the git tag.